### ISSUE TITLE
Accordions not opening in ie11, using wordpress API with React + Gatsby

### ISSUE THREAD

**Issue by tweetzal** [springload/react-accessible-accordion#224]
Hi there,
Great component firstly - thank you very much for sharing.
However, I am having issues in ie11.
The accordions dont open in ie11 ? 
I am mapping from Wordpress Rest API using advanced custom fields, see my code snippet below. Am I missing anything? 

```jsx
<div className={pageStyles.wpAccordions}>
    <Accordion
        className={pageStyles.accordion}
        allowZeroExpanded={true}
        allowMultipleExpanded={true}
    >
        {acf_field.accordions.map((acc_field, index) => {
            return (
                <AccordionItem className={pageStyles.accordionItem}>
                    <AccordionItemHeading
                        className={pageStyles.accordionItemHeading}
                    >
                        <AccordionItemButton
                            className={pageStyles.accordionItemButton}
                        >
                            {acc_field.accordion_title}
                        </AccordionItemButton>
                    </AccordionItemHeading>
                    <AccordionItemPanel
                        className={pageStyles.accordionItemPanel}
                    >
                        <div
                            key={body + index}
                            dangerouslySetInnerHTML={{
                                __html: acc_field.accordion_content,
                            }}
                        />
                    </AccordionItemPanel>
                </AccordionItem>
            );
        })}
    </Accordion>
</div>
```

**Comment by ryami333** [comment]
Hi the @tweetzal - thanks for your issue. I hope you don't mind but I edited it for you so that the code snippet was formatted and had syntax highlighting.

I can't see anything immediately obvious with your snippet, it all looks pretty sane to me. So then, in order to help you further I'm going to require a little more detail/context about your problem. When you say that the accordion items don't open, what exactly do you mean by that?

Have you inspected the markup in a console to see whether or not the HTML attributes are changing when you click on items? If they are changing but _visually_ the items are not opening and closing, then you may have a styling issue on your hands. It might be a matter of simply applying the correct `display` CSS property to the appropriate selector.

Have you inspected the console to see whether there are any Javascript errors? If so, you might not have applied the [correct polyfills](https://reactjs.org/blog/2017/09/26/react-v16.0.html#javascript-environment-requirements) required for React 16 to be compatible with IE11.

Have you actually verified that it's working in other browsers but not in IE11?

Any other information you could provide would be great - and an minimally reproducible standalone example on Codesandbox would be even better! Cheers 😄

**Comment by janzenz** [comment]
Most likely related to #237 due to missing `Array.prototype.findIndex` polyfill.

**Comment by ryami333** [comment]
Should be fixed in v3.0.1, thanks for your issue @tweetzal, please re-open or create a new issue if you're still having problems after bumping your RAA version. Cheers! 👋
